src/RE.h

```diff
@@ -89,6 +90,7 @@ class Specific_RE_Matcher {
 
     bool MatchAll(const char* s);
     bool MatchAll(const String* s);
+    bool MatchAll(std::string_view sv);
```

Comment:
Sorry to pile more onto this, but could you add overloads taking `std::string_view` for `Match` and `LongestMatch` for consistency? We should then be able to reimplement their `const char*` and `const String*` overloads in terms of `std::string_view` as well.